test/utils/blueprints/network_blueprints.hpp

```diff
@@ -358,6 +358,31 @@ mockturtle::names_view<Ntk> topolinano_network()
     return ntk;
 }
 
+template <typename Ntk>
+mockturtle::names_view<Ntk> dangling_pi_network()
+{
+    mockturtle::names_view<Ntk> ntk{};
+
+    const auto                  a = ntk.create_pi("a");
+    const auto                  b = ntk.create_pi("b");
+    const auto                  c = ntk.create_pi("c");
+    const auto                  d = ntk.create_pi("d");
+    const auto                  e = ntk.create_pi("e");
+    [[maybe_unused]] const auto f = ntk.create_pi("f");
```

Comment:
If not used, don't give it a name :wink: 
```suggestion
    ntk.create_pi("f");
```